arch/arm/boot/dts/imx6ul-ts7180.dts

```diff
@@ -0,0 +1,1020 @@
+		"ENET1_TX_DATA1",
+		"ENET1_TX_EN",
+		"ENET1_TX_CLK",
+		"ENET1_RX_ER",
+		"ENET2_RX_DATA0",
+		"ENET2_RX_DATA1",
+		"ENET2_RX_EN",
+		"ENET2_TX_DATA0",
+		"ENET2_TX_DATA1",
+		"ENET2_TX_EN",
+		"ENET2_TX_CLK",
+		"ENET2_RX_ER",
+		"SD_CMD",
+		"SD_CLK",
+		"SD_D0",
+		"SD_D1",
+		"SD_D2",
+		"SD_D3",
+		"NA",
+		"NA",
+		"NA",
+		"NA",
+		"NA",
+		"NA",
+		"NA",
+		"NA",
+		"NA",
+		"NA";
+};
+
+&gpio3 {
+	gpio-line-names =
+		"HD12_SPI_CS",
+		"JTAG_FPGA_TCK",
+		"JTAG_FPGA_TMS",
+		"JTAG_FPGA_TDI",
+		"WDOG#",
+		"I2C_3_DAT",
+		"I2C_3_CLK",
+		"NA",
+		"HD12_I2C_CLK",
+		"HD12_I2C_DAT",
+		"HD12_DIG_INPUT",
+		"NO_CHRG_JMP#",
+		"EN_NIM_USB#",
+		"CAN_1_TXD",
+		"CAN1_RXD_3V",
+		"XBEE_CTS#",
+		"U_BOOT_JMP#",
+		"NA",
+		"PUSH_SW_CPU#",
+		"NIMBEL_PWR_ON",
+		"NA",
+		"UART7_TXD",
+		"UART7_RXD",
+		"ID4",
+		"JTAG_FPGA_TDO",
+		"UART8_TXD",
+		"UART8_RXD",
+		"ID1",
+		"ETH_PHY_RESET#",
+		"NA",
+		"NA",
+		"NA";
+};
+
+&gpio4 {
+	gpio-line-names =
+		"EMMC_CLK",
+		"EMMC_CMD",
+		"EMMC_D0",
+		"EMMC_D1",
+		"EMMC_D2",
+		"EMMC_D3",
+		"SPI_4_CLK",
+		"SPI_4_MOSI",
+		"SPI_4_MISO",
+		"SPI_4_CS#",
+		"MAG_N_IRQ",
+		"FPGA_RESET#",
+		"SPI_3_FPGA_CS#",
+		"SPI_3_CLK",
+		"SPI_3_MOSI",
+		"SPI_3_MISO",
+		"PWM_5",
+		"UART6_TXD",
+		"UART6_RXD",
+		"ID5",
+		"GYRO_INT",
+		"6UL_FORCE_5V_ON",
+		"EN_EMMC_3.3V#",
+		"EN_YEL_LED#",
+		"EN_RED_LED#",
+		"EN_GRN_LED#",
+		"EN_BLU_LED",
+		"FRAM_SPI_CS#",
+		"SD_VSEL_1.8V",
+		"NA",
+		"NA",
+		"NA";
+};
+
+&gpio5 {
+	gpio-line-names =
+		"POWER_FAIL",
+		"GPS_PPS_OUT",
+		"NA",
+		"GPIO_DVFS",
+		"NA",
+		"SILAB_C2_CLK",
+		"SILAB_C2_DATA",
+		"SILAB_C2_RESET",
+		"SPARE_4",
+		"NA",
+		"NA",
+		"NA",
+		"NA",
+		"NA",
+		"NA",
+		"NA",
+		"NA",
+		"NA",
+		"NA",
+		"NA",
+		"NA",
+		"NA",
+		"NA",
+		"NA",
+		"NA",
+		"NA",
+		"NA",
+		"NA",
+		"NA",
+		"NA",
+		"NA",
+		"NA";
+};
+
+&gpc {
+	fsl,cpu_pupscr_sw2iso = <0x1>;
+	fsl,cpu_pupscr_sw = <0x0>;
+	fsl,cpu_pdnscr_iso2sw = <0x1>;
+	fsl,cpu_pdnscr_iso = <0x1>;
+	fsl,ldo-bypass = <0>; /* DCDC, ldo-enable */
+};
+
+&uart1 {
+	pinctrl-names = "default";
+	pinctrl-0 = <&pinctrl_uart1>;
+	status = "okay";
+};
+
+&uart2 {
+	pinctrl-names = "default";
+	pinctrl-0 = <&pinctrl_uart2>;
+	status = "okay";
+};
+
+&uart3 {
+	pinctrl-names = "default";
+	pinctrl-0 = <&pinctrl_uart3>;
+	status = "okay";
+	uart-has-rtscts;
+};
+
+&uart4 {
+	pinctrl-names = "default";
+	pinctrl-0 = <&pinctrl_uart4>;
+	status = "okay";
+	uart-has-rtscts;
+	rts-gpios = <&gpio1 19 GPIO_ACTIVE_HIGH>; /* EN_485 is the TX enable signal */
+	linux,rs485-enabled-at-boot-time;
+};
+
+&uart5 {
+	pinctrl-names = "default";
+	pinctrl-0 = <&pinctrl_uart5>;
+	status = "okay";
+};
+
+
+&uart6 {
+	pinctrl-names = "default";
+	pinctrl-0 = <&pinctrl_uart6>;
+	status = "okay";
+};
+
+&uart7 {
+	pinctrl-names = "default";
+	pinctrl-0 = <&pinctrl_uart7>;
+	status = "okay";
+};
+
+&uart8 {
+	pinctrl-names = "default";
+	pinctrl-0 = <&pinctrl_uart8>;
+	status = "okay";
+};
+
+&usdhc1 {
```

Comment:
Is the SD card properly wired for different UHS speed modes? This was a problem for sure on other 6ul platforms. Be sure these are supported before calling these out.